kubectl.zsh

```diff
@@ -27,7 +27,12 @@ function _zsh_kubectl_prompt_precmd() {
     fi
 
     zstyle -s ':zsh-kubectl-prompt:' updated_at updated_at
-    if ! now="$(stat -c '%y' "$kubeconfig" 2>/dev/null)"; then
+    if [[ $(uname) == "Linux" ]]; then
+        now="$(stat -c '%y' "$kubeconfig" 2>/dev/null)"
+    elif [[ $(uname) == "Darwin" ]]; then
+        now="$(stat -f '%m' "$kubeconfig" 2>/dev/null)"
+    fi
+    if ! [[ -z $now ]]; then
```

Comment:
Please remove `!` and enclose `$now` in double quotes. Here expects to return true if `$now` is empty string.

```sh
if [[ -z "$now" ]]; then
```